src/fe/src/App.js

```diff
@@ -104,6 +115,8 @@ function App() {
       newCheckbox.hierarchy = checked;
     } else if (key === "labels") {
       newCheckbox.labels = checked;
+    } else if (key === "keyword using Rovo") {
```

Comment:
유저들이 rovo 가 어떻게 link 를 만드는지는 몰라도 될거 같아요

```suggestion
    } else if (key === "rovo") {
```